data/sql_setup/prepare_schedule_a.sql

```diff
@@ -124,16 +328,100 @@ $$ language plpgsql;
 -- Create new triggers
 drop trigger if exists nml_sched_a_after_trigger on disclosure.nml_sched_a;
 create trigger nml_sched_a_after_trigger after insert or update
-    on disclosure.nml_sched_a for each row execute procedure ofec_sched_a_insert_update_queues(:START_YEAR_AGGREGATE);
+    on disclosure.nml_sched_a for each row execute procedure ofec_sched_a_insert_update(:START_YEAR_AGGREGATE);
 
 drop trigger if exists nml_sched_a_before_trigger on disclosure.nml_sched_a;
 create trigger nml_sched_a_before_trigger before delete or update
-    on disclosure.nml_sched_a for each row execute procedure ofec_sched_a_delete_update_queues(:START_YEAR_AGGREGATE);
+    on disclosure.nml_sched_a for each row execute procedure ofec_sched_a_delete_update(:START_YEAR_AGGREGATE);
 
 drop trigger if exists f_item_sched_a_after_trigger on disclosure.f_item_receipt_or_exp;
 create trigger f_item_sched_a_after_trigger after insert or update
-    on disclosure.f_item_receipt_or_exp for each row execute procedure ofec_sched_a_insert_update_queues(:START_YEAR_AGGREGATE);
+    on disclosure.f_item_receipt_or_exp for each row execute procedure ofec_sched_a_insert_update(:START_YEAR_AGGREGATE);
 
 drop trigger if exists f_item_sched_a_before_trigger on disclosure.f_item_receipt_or_exp;
 create trigger f_item_sched_a_before_trigger before delete or update
-    on disclosure.f_item_receipt_or_exp for each row execute procedure ofec_sched_a_delete_update_queues(:START_YEAR_AGGREGATE);
+    on disclosure.f_item_receipt_or_exp for each row execute procedure ofec_sched_a_delete_update(:START_YEAR_AGGREGATE);
```

Comment:
*As noted from above with the older functions.*

Should we have statements to drop the older functions at this point, where it might be safer to do so?  We'd need to have some `TODO` comments to remind us to take them out after this has been propagated across all environments, like with similar changes from before.